components/gantry/multiaxis/multiaxis.go

```diff
@@ -91,8 +91,19 @@ func newMultiAxis(
 	return mAx, nil
 }
 
+// Home runs the homing sequence of the gantry and returns true once completed.
+func (g *multiAxis) Home(ctx context.Context, extra map[string]interface{}) (bool, error) {
+	for _, subAx := range g.subAxes {
+		homed, err := subAx.Home(ctx, nil)
+		if err != nil {
+			return homed, err
+		}
+	}
+	return true, nil
```

Comment:
I was thinking the only reason it would fail is if there's an error? and in that case `homed` would be false. It will only return true if it makes it through the entire loop which would only happen if there were no failures. Or is there something else that we want to consider "failing"?

Reply:
good question, we are assuming our single-axis driver logic now, but imagine if someone implements a gantry that can fail at homing but not error and passes it into this function as a gantry (say they implement their own modular gantry and find this multi-axis driver neat). I'd explicitly return false, err for the err case and check if all have successfully homed with a true and return that instead, if that makes sense.